doc-content/enterprise-only/openshift/operator-createapp-proc.adoc

```diff
@@ -0,0 +1,154 @@
++
+[subs="attributes,verbatim,macros"]
+----
+metadata:
+  name: keystore-config
+spec:
+  environment: <environment_type>
+  objects:
+    console:
+      keystoreSecret: console-keystore
+    servers:
+      - name: server-a
+        deployments: 2
+        keystoreSecret: server-a-keystore
+      - name: server-b
+        keystoreSecret: server-b-keystore
+    smartRouter:
+      keystoreSecret: smartrouter-keystore
+----
++
+Replace `<environment_type>` with the type of environment that you want to configure.
++
+** The following snippet sets the password for the administrator user (`admin`) and the application name (`app2`):
++
+[subs="attributes,verbatim,macros"]
+----
+  commonConfig:
+    adminPassword: password
+    applicationName: app2
+----
++
+** The following snippet sets up LDAP authentication. The parameters correspond to the settings of the LdatExtended Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/login_module_reference/#ldapextended_login_module[LdapExtended Login Module]. 
++
+[subs="attributes,verbatim,macros"]
+----
+auth:
+  ldap:
+    url: ldaps://myldap.example.com
+    bindDN: uid=admin,ou=users,ou=exmample,ou=com
+    bindCredential: s3cret
+    baseCtxDN: ou=users,ou=example,ou=com
+    baseFilter: (uid={0})
+    searchScope: SUBTREE_SCOPE
+    roleAttributeID: memberOf
+    rolesCtxDN: ou=groups,ou=example,ou=com
+    roleFilter: (memberOf={1})
+    defaultRole: guest
+  roleMapper:
+    rolesProperties: /conf/roleMapper.properties
+    replaceRole: true
+----
++
+If the LDAP server does not define all the roles required for your deployment, you can map LDAP groups to {PRODUCT} roles. To enable LDAP role mapping, set the `rolesProperties` value to the fully qualified pathname of a file that defines role mapping, for example, `/opt/eap/standalone/configuration/rolemapping/rolemapping.properties`. You must provide this file and mount it at this path in all applicable deployment configurations; for instructions, see <<rolemapping-proc>>.
++
+If `replaceRole` is set to `true`, mapped roles replace the roles defined on the LDAP server; if it is set to `false`, both mapped roles and roles defined on the LDAP server are set as user application roles. The default setting is `false`.
+. After completing the modification of the YAML source, click *Create* to create the application.
+
+[NOTE]
+====
+You can view other configuration samples, for example, for servers that use differen database servers, at https://github.com/kiegroup/kie-cloud-operator/tree/1.0.x/deploy/examples 
```

Comment:
We generally don't point to outside repos, but in this case I agree because it's not something for our product, correct? Just samples? But either way, maybe truncate with linked text instead of the raw URL. Maybe "servers, see the https://...[operators samples repository] in GitHub." Or something.

Reply:
@mramendi additionally as we prepare for 7.3.1, it's becoming apparent that we should use the release tag of `1.0.0` instead of `1.0.x` ... like so:
https://github.com/kiegroup/kie-cloud-operator/tree/1.0.0/deploy/examples

Can we change this everywhere it's referenced?

Reply:
In this case it's because it is tech preview. This SHOULD be downloadable from our Portal instead, but I'm not getting into this battle for this release.

Reply:
@tchughesiv in the nick of time! changed just before a merge